wandb/sdk/artifacts/artifact.py

```diff
@@ -1894,6 +1927,11 @@ def link(self, target_path: str, aliases: Optional[List[str]] = None) -> None:
         """
         if self._state == ArtifactState.PENDING:
             raise ArtifactNotLoggedError(self, "link")
+        if self.ttl is not None:
+            wandb.termwarn(
+                "Artifact TTL will be removed for source artifacts that are linked to portfolios."
```

Comment:
Nit: it sounds like we will remove the TTL at some point in the future, not immediately. How about:
```suggestion
                "Artifacts linked to a portfolio cannot have a TTL. The artifact will no longer have a TTL."
```